bitnami/kubeapps/Chart.yaml

```diff
@@ -25,4 +25,4 @@ maintainers:
 name: kubeapps
 sources:
   - https://github.com/kubeapps/kubeapps
-version: 7.1.8
+version: 7.2.0
```

Comment:
Mmm, wondering now if, given that it is a backwards uncompatiblke change (<=v.1.15), perhaps it's a major version bump. But let the Bitnami team decide here.

Reply:
Could you give more context @antgamdia about the incompatibility?
My main question is: what happens if I install the chart version `7.1.8` and then I upgrade to `7.2.0`? Would that break my installation?

Reply:
Well... it depends :grimacing:  Given that Kubeapps is supported in k8s 1.16 onwards, if only Helm were able to perform CRD upgrades (which is not https://github.com/helm/helm/issues/7735) it wouldn't be a breaking change.
What's more, if the chart were to be installed using Flux instead of the Helm client (https://fluxcd.io/docs/components/helm/helmreleases/#crds), Flux would handle the upgrade, so, it wouldn't affect our target users.

So: is it a breaking change for Helm client users, yes, it is. We need to add a message in the readme.md, or even also in the upgrade message (as we do when the postgresql password is not being provided during an upgrade).

An additional thought on this: should we hold this PR off until we also add the new `kubeappsapis` service in the chart so that we only release a single major version bump?   [yes, @juan131 , we will need your team to create yet another kubeapps asset, but this time is Go, not Rust :stuck_out_tongue_closed_eyes: ]